instance/static/js/src/openedx_appserver.js

```diff
@@ -59,13 +59,27 @@ app.controller("OpenEdXAppServerDetails", ['$scope', '$state', '$stateParams', '
                 $scope.$parent.refresh().then(function() {
                     $scope.refresh();
                 });
-                $scope.notify($scope.appserver.name + ' is now active. The DNS changes may take a while to propagate.');
+                $scope.notify($scope.appserver.name + ' is now active. The load balancer changes will take a short while to propagate.');
             }, function() {
                 $scope.refresh();
                 $scope.notify('An error occurred. ' + $scope.appserver.name + ' could not be made active.', 'alert');
             });
         };
 
+        $scope.make_appserver_inactive = function() {
```

Comment:
This function is almost identical to `make_appserver_active`.  Not sure whether it makes sense to factor out the common bits or whether the code becomes too awkward if we do.  You probably already thought about this, so what do you think?